README.md

```diff
@@ -77,6 +77,10 @@ custom:
 
     # Enable tracing on Lambda function using dd-trace, datadog's APM library. Requires datadog log forwarder to be set up. Defaults to true.
     enableDDTracing: true
+
+    # When set, the plugin will subscribe the lambda's cloudwatch log groups to the forwarder with the
+    # given arn. Don't use this if you are already using Datadog's automatic trigger management.
```

Comment:
Does this conflict with the automatic trigger management somehow? Otherwise I would delete _"Don't use this if you are already using Datadog's automatic trigger management."_ In fact, I would treat the automatic trigger management as a "catch all" or "fallback" solution for ad-hoc functions, and for functions created by the serverless framework i'd prefer have the trigger defined in serverless.yml (even when automatic trigger management is configured).

Reply:
Yeah, it can conflict. Essentially, if you are using automatic trigger management, and it's already subscribed to your function's log group, then trying to set the subscription will fail, since you can't have two subscriptions. I actually ran into this issue during testing. For new functions it's fine though.

Reply:
I've reworded this section, and added a check before we try to apply the subscription to the arn, so it's less likely a conflict will occur.